### PR TITLE
Select by categories to show in the inventory

### PR DESCRIPTION
**Pull request by Ivans51**
Signed-off-by: Ivan Del Pino <idelpino@teclib.com>

### Changes description

Added view and button to show lists of categories
Filter to categories in report view

### Checklist

Please check if your PR fulfills the following specifications:

- [x] Tests for the changes have been added
- [ ] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@ivans51|12|

### References

Closes: https://github.com/flyve-mdm/android-inventory-agent/issues/212
Related #N/A
Depends on #N/A

### CODE REVIEW

File: app/src/main/java/org/flyve/inventory/agent/core/categories/Categories.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * Flyve MDM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * ------------------------------------------------------------------------------
+ * @author    Ivan Del Pino
```
Comment (ajsb85) [inline app/src/main/java/org/flyve/inventory/agent/core/categories/Categories.java]: Hi, @Ivans51 
Don't forget your email address.

File: app/src/main/java/org/flyve/inventory/agent/adapter/CategoriesAdapter.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * Flyve MDM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * ------------------------------------------------------------------------------
+ * @author    Ivan Del Pino
```
Comment (ajsb85) [inline app/src/main/java/org/flyve/inventory/agent/adapter/CategoriesAdapter.java]: Hi, @Ivans51 
Add your email address

Comment (Ivans51) [reply app/src/main/java/org/flyve/inventory/agent/adapter/CategoriesAdapter.java]: Ready

File: app/src/main/java/org/flyve/inventory/agent/core/categories/CategoriesModel.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * Flyve MDM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * ------------------------------------------------------------------------------
+ * @author    Ivan Del Pino
```
Comment (ajsb85) [inline app/src/main/java/org/flyve/inventory/agent/core/categories/CategoriesModel.java]: @Ivans51 Your email Email

File: app/src/main/java/org/flyve/inventory/agent/core/categories/CategoriesPresenter.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * Flyve MDM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * ------------------------------------------------------------------------------
+ * @author    Ivan Del Pino
```
Comment (ajsb85) [inline app/src/main/java/org/flyve/inventory/agent/core/categories/CategoriesPresenter.java]: @Ivans51 Your email Email

File: app/src/main/java/org/flyve/inventory/agent/core/categories/Categories.java (hunk 1 above)
Comment (ajsb85) [reply app/src/main/java/org/flyve/inventory/agent/core/categories/Categories.java]: @Ivans51 Your email Email

File: app/src/main/java/org/flyve/inventory/agent/adapter/CategoriesAdapter.java [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * Flyve MDM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * ------------------------------------------------------------------------------
+ * @author    Ivan Del Pino - <idelpino@teclib.com>
+ * @copyright Copyright Teclib. All rights reserved.
```
Comment (ajsb85) [inline app/src/main/java/org/flyve/inventory/agent/adapter/CategoriesAdapter.java]: @copyright Copyright © 2018 Teclib'

File: app/src/main/java/org/flyve/inventory/agent/core/categories/Categories.java [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+ * @copyright Copyright Teclib. All rights reserved.
```
Comment (ajsb85) [inline app/src/main/java/org/flyve/inventory/agent/core/categories/Categories.java]: @copyright Copyright © 2018 Teclib'

File: app/src/main/java/org/flyve/inventory/agent/core/categories/CategoriesModel.java [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+ * @copyright Copyright Teclib. All rights reserved.
```
Comment (ajsb85) [inline app/src/main/java/org/flyve/inventory/agent/core/categories/CategoriesModel.java]: @copyright Copyright © 2018 Teclib'

File: app/src/main/java/org/flyve/inventory/agent/core/categories/CategoriesPresenter.java [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+ * @copyright Copyright Teclib. All rights reserved.
```
Comment (ajsb85) [inline app/src/main/java/org/flyve/inventory/agent/core/categories/CategoriesPresenter.java]: @copyright Copyright © 2018 Teclib'

File: app/src/main/java/org/flyve/inventory/agent/ui/ActivityCategories.java [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * Flyve MDM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * ------------------------------------------------------------------------------
+ * @author    Ivan Del Pino - <idelpino@teclib.com>
+ * @copyright Copyright Teclib. All rights reserved.
```
Comment (ajsb85) [inline app/src/main/java/org/flyve/inventory/agent/ui/ActivityCategories.java]: `@copyright Copyright © 2018 Teclib'`

**Comment by Ivans51** [comment]
Updated the header of the docs

**Pull request merged**